src/qml/LayerTreeItemProperties.qml

```diff
@@ -44,6 +46,7 @@ Popup {
     expandCheckBox.text = layerTree.data( index, FlatLayerTreeModel.Type ) === 'group' ? qsTr('Expand group') : qsTr('Expand legend item')
     expandCheckBox.checked = !layerTree.data(index, FlatLayerTreeModel.IsCollapsed)
 
+    reloadDataButtonVisible = true// layerTree.data(index, FlatLayerTreeModel.CanReloadData)
```

Comment:
You want to make it depending on `CanReloadData`, no?

Reply:
right, that is a debug leftover, thanks !

Reply:
```suggestion
    reloadDataButtonVisible = layerTree.data(index, FlatLayerTreeModel.CanReloadData)
```